atomics/include/desul/atomics/Compare_Exchange_SYCL.hpp

```diff
@@ -13,7 +13,12 @@ SPDX-License-Identifier: (BSD-3-Clause)
 #include "desul/atomics/SYCLConversions.hpp"
 #include "desul/atomics/Common.hpp"
 
+// FIXME_SYCL
```

Comment:
Please add a comment on the SYCL minimum version requirement

Reply:
There is no version requirement. Older compiler only provide `CL/sycl.hpp` but the standard says this should be `sycl/sycl.hpp`. Hence, we choose the latter when the former is not available.